frontend/uddhar-frontend/src/volunteer/components/OrganizationDetails.jsx

```
@@ -1,45 +1,86 @@
+            target="_blank"
+            rel="noopener noreferrer"
+            className="text-blue-500 hover:underline"
+          >
+            Social Media
+          </a>
+        </p>
+        <div className="mt-6 flex justify-center">
+          <button
+            className="px-6 py-2 bg-gray-600 text-white rounded-lg hover:bg-gray-700 transition-colors"
+            onClick={() => setSelectedOrganization(null)}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)